pkg/controller/spinnakerservice/spinnakerservice_controller.go

```diff
@@ -68,7 +69,14 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	// Watch for potential object owned by SpinnakerService
-	return c.Watch(&source.Kind{Type: &appsv1.Deployment{}}, &handler.EnqueueRequestForOwner{
+	err = c.Watch(&source.Kind{Type: &appsv1.Deployment{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    TypesFactory.NewService(),
+	})
+	if err != nil {
+		return err
+	}
+	return c.Watch(&source.Kind{Type: &corev1.Service{}}, &handler.EnqueueRequestForOwner{
```

Comment:
If I get that right, it should mean users lose the ability to edit service after the fact? (e.g. to change the type). It's probably okay since we now support patches but worth calling out in the change log

Reply:
That's right when we're exposing spinnaker. Although I haven't tested the scenario, if you omit the `expose` section entirely you can still make manual changes to the services, operator will enter the reconciliation loop, look that expose is disabled and will not touch the services. However, manifests generated by halyard may override external changes anyway, so using patches is the way to get them customized.